### DISCUSSION TITLE
Question about submitting

### DISCUSSION THREAD

**Discussion by makwingchi** [PaddlePaddle/PaddleSpatial#67] [BaiduKDDCup2022] [answered]
I just used the baseline code you provided (the ones under test-1.zip) to test the submission system, however, the following error was returned - The preparation script, i.e. 'prepare.py', does NOT exist! The file structure in the submitted zip file is as follows:

paddlepaddle
|- \_\_init\_\_.py
|- checkpoints
|- predict.py
|- prepare.py
|- ...

I wonder whether you could provide any insights to this problem.

Thanks!

**Comment by ramber1836** [comment, marked as answer]
Please refer to this discussion
https://github.com/PaddlePaddle/PaddleSpatial/discussions/65
